gui/src/2D/advancedDynamicTexture.ts

```diff
@@ -501,15 +501,6 @@ export class AdvancedDynamicTexture extends DynamicTexture {
         var renderWidth = engine.getRenderWidth() * this._renderScale;
         var renderHeight = engine.getRenderHeight() * this._renderScale;
 
-        if (this._renderAtIdealSize) {
```

Comment:
Can you elaborate more on why you remove that code?

Reply:
This line was changing the render w/h but then it appearing to be re-accounted for other file. Calculating it twice?

Reply:
but this needs to happen as the goal of renderAtIdealSize is precisely to do that scale (check the doc link I shared)